src/main/kotlin/no/nav/faktureringskomponenten/service/FakturaGenerator.kt

```diff
@@ -0,0 +1,84 @@
+package no.nav.faktureringskomponenten.service
+
+import no.nav.faktureringskomponenten.domain.models.Faktura
+import no.nav.faktureringskomponenten.domain.models.FakturaLinje
+import no.nav.faktureringskomponenten.domain.models.FakturaserieIntervall
+import no.nav.faktureringskomponenten.domain.models.FakturaseriePeriode
+import java.time.LocalDate
+import java.time.temporal.IsoFields
+import java.time.temporal.TemporalAdjusters
+
+open class FakturaGenerator(private val fakturalinjeGenerator: FakturalinjeGenerator = FakturalinjeGenerator()) {
```

Comment:
Nå har vi både mapper og generator klasse, trenger vi det? Hvis jeg skal hente frem et eksempel, ser jeg i API at i f.eks behandlingService klassen lages det behandlinger, kanskje vi kan døpe om mapper og generator til Service?

Reply:
Jeg kan være enig i det. Er usikker på dette pga. ganske minimal logikk

Reply:
Men det er logikk, tror det er best for å forhindre forvirringer.

Reply:
Ok, jeg flytter og endre navn også. Jeg kan se at det blir mer konsekvent